mne/viz/_3d.py

```diff
@@ -353,12 +353,55 @@ def plot_trans(info, trans_fname='auto', subject=None, subjects_dir=None,
     return fig
 
 
+def _percent_to_control_points(clim, stc_data, colormap):
+    """Private helper function to convert percentiles to control points.
+
+    Note: If using 'mne_analyze', generate cmap control points for a directly
+    mirrored cmap for simplicity (i.e., no normalization is computed to account
+    for a 2-tailed mne_analyze cmap).
+
+    Parameters
+    ----------
+    clim : str | dict
+        Desired percentages used to set cmap control points.
+
+    Returns
+    -------
+    ctrl_pts : list (length 3)
+        Array of floats corresponding to values to use as cmap control points.
+    """
+
+    # Based on type of limits specified, get cmap control points
+    if clim == 'auto':
+        # Set upper and lower bound based on percent, and get average between
+        ctrl_pts = np.percentile(np.abs(stc_data), [96, 99.95])
+        ctrl_pts.insert(1, np.average(ctrl_pts))
+    elif isinstance(clim, dict):
+        # Get appropriate key for clim if it's a dict
+        limit_key = ['lims', 'pos_lims'][colormap == 'mne_analyze']
```

Comment:
If you think it's needed, I could... It's pretty clear in the documentation and the function will choose the appropriate one if both are defined.